src/main/java/io/seqera/tower/cli/commands/computeenvs/platforms/AwsBatchForgePlatform.java

```diff
@@ -99,7 +107,7 @@ private ForgeConfig buildForge() {
                 .type(provisioningModel)
                 .maxCpus(maxCpus)
                 .ebsAutoScale(!noEbsAutoScale)
```

Comment:
I you don't set `--no-ebs-auto-scale` flag then you always get an error from API. I think that the logic should be to mimic as much as possible same behavior that we have at the web interface. On the web the user only has to select Fusion v2 and then wave is set to enabled and EBS autoscale is disabled automatically.

The Wave requirement it's a bit different because it is already possible to use Fusion without Wave, but AFAIK it's still not supported on the web side. Even when it is supported is not going to be the default, so the user will need to do a conscious decision when selecting Fusion to then disable Wave, the default would still be enabled.

So my suggestion would be:

- User that wants to use Fusion v2 then just `--fusion-v2` is enough (wave is enabled and EBS autoscale disabled).  
- User that wants to use Fusion v2 without Wave then `--fusion-v2 --no-wave` is needed (still EBS autoscale disabled).
- User that only wants to use Wave then `--wave` is needed and no extra logic is required.

What do you think @pditommaso @drpatelh ?

PD: I agree that some of this logic maybe better at the API side. But this is not what we de now, we are encoding this logic on the web frontend, so I think that CLI is just another frontend of the API and should mimic web logic as much as possible.

Reply:
Alternative suggestion (because I don't like to use the `v2` suffix).

We can add `--fusion-wave` flag and `--fusion-no-wave` flag to cover this two use cases.

Adding also @gwright99 for opinions.

Reply:
why options with mixed semantics ie. `--fusion-wave` and `--fusion-no-wave` and why negated ones `-no-ebs-auto-scale` and `--no-wave`?

Reply:
Negated options are because you want to set the oposite as the default behavior (as we are doing at the website)

Reply:
> as we are doing at the website

what do you mean?

Reply:
I mean that by default at the web form the EBS autoscale is enabled. You need a conscious decision of the user to disable it (in the CLI this means a `--no-ebs-autoscale` option). 

This is the philosophy that is following all the CLI; if the user do not add a specific flag then all the defaults and behavior will be same as we have at the website form.

With Fusion happens something similar, when the user enables Fusion then Wave is automatically enabled.

Reply:
it's a -1 for me. They should be boolean options `--wave false|true`, `--fusion-v2 false|true`. Mixed options should be avoid e.g. `-fusion-wave` just blow up the number of option for no reason

Reply:
I think that nowadays it is more the facto standard to use `--OPTION` and `--no-OPTION` for boolean flags. But also, more important than that, (as [this](https://clig.dev/) guidelines suggest) is to keep consistency along your CLI.

And right now we are already using the option/no-option approach.

Reply:
I see only one.  

```
» tw compute-envs add aws-batch forge -h | grep -- --no
      --no-ebs-auto-scale                        Disable the provisioning of EBS auto-expandable disk.
```

IMO makes the code and validation much more complex

Reply:
As discussed in #330 I'm going to leave `--no-ebs-auto-scale` and rely on Tower API parameter validation.